game-app/game-core/src/main/java/tools/map/making/ui/MapMakingPanelBuilder.java

```diff
@@ -0,0 +1,48 @@
+package tools.map.making.ui;
+
+import java.awt.Component;
+import java.util.List;
+import javax.swing.Box;
+import javax.swing.BoxLayout;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import org.triplea.swing.JButtonBuilder;
+
+public class MapMakingPanelBuilder {
+  protected static final int SPACING_HEIGHT = 30;
+
+  final JPanel panel = new JPanel();
+
+  public record ButtonSpec(String labelText, Runnable runnable) {}
+
+  public MapMakingPanelBuilder(String title) {
```

Comment:
Recommendation to clean up API:

- convert this method to static method (and rename it to something like 'build' or something; we will be transitioning to a factory method)
- return JPanel
- args: `String title, JButton ... buttons`